utils_nlp/models/transformers/common.py

```diff
@@ -223,15 +165,10 @@ def move_batch_to_device(batch, device):
                 if (step + 1) % gradient_accumulation_steps == 0:
```

Comment:
Also maybe the gradient clipping should be done after accumulating gradients like https://github.com/huggingface/transformers/blob/dfe012ad9d6b6f0c9d30bc508b9f1e4c42280c07/examples/run_glue.py#L228